src/unix/tty.c

```diff
@@ -64,7 +64,11 @@ static int isreallyatty(int file) {
 
 static int orig_termios_fd = -1;
 static struct termios orig_termios;
+#if defined(UV_PREFER_UNFAIR_LOCK)
+os_unfair_lock termios_unfair_lock = OS_UNFAIR_LOCK_INIT;
```

Comment:
Question: is this necessary? The spinlock is there because node.js calls `uv_tty_reset()` from a signal handler on program exit. Lock contention is otherwise not a concern, it's never contended during normal program execution.

Reply:
We have reverted it